api/controllers/healthcheck.js

```diff
@@ -0,0 +1,19 @@
+const { createPool } = require('mariadb');
+const config = require('../config');
+const lib = require('../lib');
+
+exports.db = async (req, res) => {
+  let conn;
+  try {
+    const pool = createPool({ ...config.mysql, connectTimeout: 3000 });
+    conn = await pool.getConnection();
+    const q = `SELECT 1 as ok FROM Verse WHERE ID=1 LIMIT 1`;
+    const rows = await conn.query(q, []);
```

Comment:
is empty array in the second argument needed?

Reply:
I guess not